test/fixtures/node_modules/pkgexports/package.json

```diff
@@ -2,6 +2,7 @@
   "exports": {
     ".": "./asdf.js",
     "./asdf": "./asdf.js",
+    "./valid-cjs": "./asdf.js",
     "./sub/": "./"
```

Comment:
```suggestion
    "./sub/": "./",
    "./encoding": "./%61sdf.js"
```

Reply:
Heh, sorry. Forgot to push properly. I added a test case with `"./space": "./sp%20ce.js"`.